.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
   "singleQuote": true,
-  "printWidth": 80
+  "printWidth": 80,
+  "trailingComma": "none",
+  "semi": false
 }
```

Comment:
I added this because my local was making a lot of changes to files I was working on